data/mods/blazemod/modinfo.json

```diff
@@ -3,11 +3,10 @@
     "type": "MOD_INFO",
     "ident": "blazemod",
     "name": "Vehicle Additions Pack",
-    "authors": [ "Blaze-whatdoyoumeanitsalreadytaken" ],
-    "maintainers": [ "Blaze-whatdoyoumeanitsalreadytaken" ],
+    "authors": [ "Blaze" ],
```

Comment:
Probably should not change author name used by original author, even if it's weird.

Reply:
Oh was that the original? Ah, looking through the history someone else changed it to reflect his github name. I'll change it back.